modules/nw-metallb-how-work-together.adoc

```diff
@@ -0,0 +1,21 @@
+[id="nw-metallb-how-work-together_{context}"]
+= How MetalLB and the MetalLB Operator work together
+
+When you add the MetalLB Operator to your cluster, you deploy a pod as part of the `metallb-operator-controller-manager` deployment.
+This controller pod monitors its own namespace for two custom resources:
+
+`MetalLB`::
+When you add an a `MetalLB` custom resource to the cluster, the MetalLB Operator deploys MetalLB on the cluster.
+Only a single instance of the custom resource is supported.
+If the instance is deleted, the Operator removes MetalLB from the cluster.
+
+`AddressPool`::
+MetalLB requires one or more pools of IP addresses that it can assign to a service when a service of type `LoadBalancer` is requested.
+When you add an `AddressPool` custom resource to the cluster, the MetalLB Operator configures MetalLB so that it can assign IP addresses from the pool.
+An address pool includes a list of IP addresses.
+The list can be a single IP address, a range specified in CIDR notation, a range specified as a starting and ending address separated by a hyphen, or a combination of the three.
+An address pool requires a name.
+The documentation uses names like `doc-example`, `doc-example-reserved`, and `doc-example-ipv6`.
+An address pool specifies whether MetalLB can automatically assign IP addresses from the pool or whether the IP addresses are reserved for services that explicitly specify the pool by name.
```

Comment:
do you wanted to hint annotation is used to select addresspool by name ?

Reply:
It's not necessarily ironclad, but my goal was to cover the properties of the address pool and leave how to use the address pool for another section.

FWIW, my anecdotal experience is that I was able to receive and IP from an address pool by specifying `spec.loadBalancerIP` and omitting the annotation. I'm not documenting that because it wasn't tested.